docs/user/pings/ping-schedules-and-timings.md

```diff
@@ -0,0 +1,21 @@
+# Ping schedules and timings overview
+
+Full reference details about the [`metrics`](metrics.html) and [`baseline`](baseline.html) ping schedules are detailed elsewhere.
+
+The following diagram shows a typical timeline of a mobile application, when pings are sent and what timing-related information is included.
+
+![ping timeline diagram](ping_timing.svg)
+
+There are two distinct runs of the application, where the OS shutdown the application at the end of Run 1, and the user started it up again at the beginning of Run 2.
+
+There are three distinct foreground sessions, where the application was visible on the screen and the user was able to interact with it.
+
+The rectangles for the baseline and metrics pings represent the "ping lifetimes" of those pings, which always start exactly at the end of the preceding ping.  The `ping_info.start_time` and `ping_info.end_time` metrics included in these pings correspond to these ping lifetimes.
+
+The `baseline.duration` metric (included only in baseline pings) corresponds to the session length, which, since ping lifetimes always extend to the next ping, is not always the same thing as the `baseline` ping's lifetime.
+
+The `submission_timestamp` is the time the ping was received at the telemetry endpoint.  It is not exactly the same as `ping_info.end_time`, since there may be various networking and system latencies (represented by the dotted horizontal line, not to scale).  Also of note is that `start_time`/`end_time` are measured using the client's real-time clock in its local timezone, which may include clock skew or completely incorrect values.
```

Comment:
```suggestion
The `submission_timestamp` is the time the ping was received at the telemetry endpoint, added by the ingestion pipeline.  It is not exactly the same as `ping_info.end_time`, since there may be various networking and system latencies (represented by the dotted horizontal line, not to scale).  Also of note is that `start_time`/`end_time` are measured using the client's real-time clock in its local timezone, which may include clock skew or completely correct values.
```

Was "completely incorrect" intentional?
I'm also wondering if we should be a bit more formal and precise about the latencies we know exist. Chris [wrote about that](https://chuttenblog.wordpress.com/2016/11/10/data-science-is-hard-case-study-latency-of-firefox-crash-rates/) a long time ago (i.e. "latency = reporting delay + submission delay + aggregation delay + migration delay + query scheduling delay"), while we were reducing latency in Firefox. Maybe there's an opportunity to make it part of our docs and reference it here?

Reply:
That's a good idea.  Since I think this is the only place in the docs that talk about submission_timestamp, we might as well do that here.

Reply:
My intention here is to say that the local clock may have a small amount of skew from universal time, or it may be wildly off -- as we see in the number of pings that are still in the 1970's.  But maybe that distinction doesn't matter -- we just need to say it's generally not 100% unreliable.

Reply:
> 
> 
> My intention here is to say that the local clock may have a small amount of skew from universal time, or it may be wildly off 

Oh! I didn't understand that, it's clearer now. Maybe change this to "or be completely incorrect?"? Or even leave it as it is, it might be just me not understanding this on a first read :)

Reply:
I decided to say "it's not a fully reliable source of time."  The details of why is probably beyond the scope of this document.

As an aside, we may still want to do something like this someday: https://www.chromium.org/developers/design-documents/sane-time